runtime/src/lib.rs

```diff
@@ -288,6 +289,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
+	// TODO: Shouldn't this be at least 1?
```

Comment:
@enthusiastmartin maybe 1000 like duster?

Reply:
Yes, we can set all ED to 1000.  

The only is that there is not dust handler - so the dust will be lost at the moment.

Reply:
Send to treasury?